notebooks/official/training/multi_node_ddp_nccl_vertex_training_with_custom_container.ipynb

```diff
@@ -454,11 +400,9 @@
         "id": "12c03eca5908"
       },
       "source": [
-        "# Create Custom Training Python Package\n",
-        "\n",
-        "Before you can perform local training, you must create source code file, requirements file, docker file.\n",
+        "## Create training application\n",
         "\n",
-        "You will create a directory and write all of our files into that folder."
+        "Before you can perform local training, you create source code file, requirements file, and docker file for your training application."
```

Comment:
Delete "you" (second one) and add "a" to the file noun phrases:
- Before you can perform local training, create a source code file, a requirements file, and a docker file for your training application.